### PR TITLE
Refactoring to explicit UI5 CLI usage

### PR DESCRIPTION
**Pull request by petermuessig**

### CODE REVIEW

File: step-by-step.md [hunk 1]
```diff
@@ -262,155 +201,254 @@ ESLint needs to be told which plug-ins to use and which JavaScript language leve
...
+As seen in the initial content of the `ui5.yaml` after running `ui5 init` only the first four lines are strictly required to use the UI5 Tooling, the rest is still useful: the `framework` section downloads the UI5 framework along with needed libraries and provides it at the virtual path `/resources` when `ui5 serve` is called. This path is from where the index.html file loads UI5. More information about the UI5 Tooling can be found here: [https://sap.github.io/ui5-tooling](https://sap.github.io/ui5-tooling).
 
-What you can do now: compile the full app into the "webapp" folder and run it from there using the UI5 tools:
+The UI5 Tooling commands require at least a `manifest.json` (an [Application Descriptor](https://sapui5.hana.ondemand.com/sdk/#/topic/be0cf40f61184b358b5faedaec98b2da.html)) in the `webapp` folder. A very simple and lightweigt `manifest.json` providing the `id`, `type`, and `version` of your application:
```
Comment (akudev) [inline step-by-step.md]: lightweight

but don't fix before merging

File: ui5.yaml [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-      debug: true
-      extraExts: "xml,json,properties"
-      port: 35729
-      path: "webapp"
+    - name: ui5-tooling-transpile-middleware
+      afterMiddleware: compression
+      configuration:
+        <<: *cfgTranspile
+    - name: ui5-middleware-livereload
+      afterMiddleware: compression
```
Comment (akudev) [inline ui5.yaml]: Somehow using the UI5 tools only ADDS complexity.
Here some complex configuration to be added which normal people will hardly write easily without looking it up each time. 
.babelrc still needed, even longer. Only package.json gets shortened, but the stuff that is removed is mostly calls to Babel, which are "standard". Seems like we are just replacing _standard_ tools with _proprietary_ tools here. Not sure this is so much better, in particular for teaching how the setup works.
I think I'll add the previous setup to a "back to basics" branch of its own.

**Review by akudev — APPROVED** [review]
As written in a code comment: I don't see this change making things much SIMPLER. :-/

File: ui5.yaml (hunk 2 above)
Comment (petermuessig) [reply ui5.yaml]: The `.babelrc` isn't needed at all! I just showcase it how to use it to not completely rewrite the hello world sample

**Comment by petermuessig** [comment]
Funnywise, if I just delete the `.babelrc` the project configuration is much smaller and simpler than it ever was before 😉

File: package.json [hunk 3]
```diff
@@ -5,40 +5,26 @@
...
   "scripts": {
-    "build": "npm-run-all build:ts build:ui5",
-    "build:opt": "npm-run-all build:ts build:ui5:opt",
-    "build:ts": "babel src --out-dir webapp --source-maps true --extensions \".ts,.js\" --copy-files",
-    "build:ui5": "ui5 build --clean-dest",
-    "build:ui5:opt": "ui5 build self-contained --clean-dest --all",
-    "start": "npm-run-all --parallel watch:ts start:ui5",
-    "watch:ts": "babel src --out-dir webapp --source-maps true --extensions \".ts,.js\" --copy-files --watch",
-    "start:ui5": "ui5 serve --port 8080 -o index.html",
+    "build": "ui5 build --clean-dest",
```
Comment (petermuessig) [inline package.json:8]: Looks so clean here! 😉

File: package.json [hunk 4]
```diff
@@ -5,40 +5,26 @@
...
-    "eslint": "^8.20.0",
-    "npm-run-all": "^4.1.5",
-    "typescript": "^4.7.4",
-    "ui5-middleware-livereload": "^0.5.13"
-  },
-  "ui5": {
-    "dependencies": [
-      "ui5-middleware-livereload"
-    ]
+    "@types/openui5": "1.112.0",
```
Comment (petermuessig) [inline package.json:20]: big reduction of necessary deps

File: ui5.yaml [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
-    afterMiddleware: compression
-    configuration:
-      debug: true
-      extraExts: "xml,json,properties"
-      port: 35729
-      path: "webapp"
+    - name: ui5-tooling-transpile-middleware
+      afterMiddleware: compression
+    - name: ui5-middleware-livereload
+      afterMiddleware: compression
```
Comment (petermuessig) [inline ui5.yaml:22]: look at the reduction of all tooling extensions - isn't that nice now how simple it is here? no config for the transpile and the livereload anymore

**Comment by petermuessig** [comment]
TBH, this looks much simpler now than the Babel setup before! And it also positions our UI5 Tooling in a nice spotlight!

**Pull request merged**